common/djangoapps/enrollment/views.py

```diff
@@ -649,6 +658,17 @@ def post(self, request):
                     enrollment_attributes=enrollment_attributes
                 )
 
+                course = get_course_by_id(course_id)
```

Comment:
Please wrap it in a try/except blovk and log results. Reference: https://github.com/raccoongang/edx-platform/blob/ginkgo-rg/lms/djangoapps/instructor/views/api.py#L545

Reply:
done